code/datums/status_effects/debuffs.dm

```diff
@@ -98,43 +98,42 @@
 	icon_state = "asleep"
 
 //STASIS
-/datum/status_effect/incapacitating/stasis
-        id = "stasis"
```

Comment:
I'd like to apologize for my past self's use of 8 spaces as the indentation for this code. I have no clue how I managed to do that.